proxysql/dockerdir/entrypoint.sh

```diff
@@ -6,70 +6,73 @@ PROXY_CFG=/etc/proxysql/proxysql.cnf
+sed "s/CLUSTER_PASSWORD='admin'/CLUSTER_PASSWORD='${OPERATOR_PASSWORD_ESCAPED:-operator}'/g" ${PROXY_ADMIN_CFG} 1<>${PROXY_ADMIN_CFG}
+sed "s/CLUSTER_PORT='3306'/CLUSTER_PORT='${CLUSTER_PORT:-3306}'/g" ${PROXY_ADMIN_CFG} 1<>${PROXY_ADMIN_CFG}
+sed "s/MONITOR_USERNAME='monitor'/MONITOR_USERNAME='monitor'/g" ${PROXY_ADMIN_CFG} 1<>${PROXY_ADMIN_CFG}
+sed "s/MONITOR_PASSWORD='monitor'/MONITOR_PASSWORD='${MONITOR_PASSWORD_ESCAPED:-monitor}'/g" ${PROXY_ADMIN_CFG} 1<>${PROXY_ADMIN_CFG}
 set -o xtrace
 
 ## SSL/TLS support
 CA=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
 if [ -f "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt" ]; then
-    CA=/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
+	CA=/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
 fi
 SSL_DIR=${SSL_DIR:-/etc/proxysql/ssl}
 if [ -f "${SSL_DIR}/ca.crt" ]; then
-    CA=${SSL_DIR}/ca.crt
+	CA=${SSL_DIR}/ca.crt
 fi
 SSL_INTERNAL_DIR=${SSL_INTERNAL_DIR:-/etc/proxysql/ssl-internal}
 if [ -f "${SSL_INTERNAL_DIR}/ca.crt" ]; then
-    CA=${SSL_INTERNAL_DIR}/ca.crt
+	CA=${SSL_INTERNAL_DIR}/ca.crt
 fi
 
 KEY=${SSL_DIR}/tls.key
 CERT=${SSL_DIR}/tls.crt
 if [ -f "${SSL_INTERNAL_DIR}/tls.key" ] && [ -f "${SSL_INTERNAL_DIR}/tls.crt" ]; then
-    KEY=${SSL_INTERNAL_DIR}/tls.key
-    CERT=${SSL_INTERNAL_DIR}/tls.crt
+	KEY=${SSL_INTERNAL_DIR}/tls.key
+	CERT=${SSL_INTERNAL_DIR}/tls.crt
 fi
 
 if [ -f "$CA" ] && [ -f "$KEY" ] && [ -f "$CERT" ] && [ -n "$PXC_SERVICE" ]; then
-    sed "s^have_ssl=false^have_ssl=true^"                   ${PROXY_CFG} 1<> ${PROXY_CFG}
-    sed "s^ssl_p2s_ca=\"\"^ssl_p2s_ca=\"$CA\"^"             ${PROXY_CFG} 1<> ${PROXY_CFG}
-    sed "s^ssl_p2s_ca=\"\"^ssl_p2s_ca=\"$CA\"^"             ${PROXY_CFG} 1<> ${PROXY_CFG}
-    sed "s^ssl_p2s_key=\"\"^ssl_p2s_key=\"$KEY\"^"          ${PROXY_CFG} 1<> ${PROXY_CFG}
-    sed "s^ssl_p2s_cert=\"\"^ssl_p2s_cert=\"$CERT\"^"       ${PROXY_CFG} 1<> ${PROXY_CFG}
+	sed "s^have_ssl=false^have_ssl=true^" ${PROXY_CFG} 1<>${PROXY_CFG}
+	sed "s^ssl_p2s_ca=\"\"^ssl_p2s_ca=\"$CA\"^" ${PROXY_CFG} 1<>${PROXY_CFG}
+	sed "s^ssl_p2s_ca=\"\"^ssl_p2s_ca=\"$CA\"^" ${PROXY_CFG} 1<>${PROXY_CFG}
+	sed "s^ssl_p2s_key=\"\"^ssl_p2s_key=\"$KEY\"^" ${PROXY_CFG} 1<>${PROXY_CFG}
+	sed "s^ssl_p2s_cert=\"\"^ssl_p2s_cert=\"$CERT\"^" ${PROXY_CFG} 1<>${PROXY_CFG}
 fi
 
 if [ -f "${SSL_DIR}/tls.key" ] && [ -f "${SSL_DIR}/tls.crt" ]; then
-    cp "${SSL_DIR}/tls.key" /var/lib/proxysql/proxysql-key.pem
-    cp "${SSL_DIR}/tls.crt" /var/lib/proxysql/proxysql-cert.pem
+	cp "${SSL_DIR}/tls.key" /var/lib/proxysql/proxysql-key.pem
+	cp "${SSL_DIR}/tls.crt" /var/lib/proxysql/proxysql-cert.pem
 fi
 if [ -f "${SSL_DIR}/ca.crt" ]; then
-    cp "${SSL_DIR}/ca.crt" /var/lib/proxysql/proxysql-ca.pem
+	cp "${SSL_DIR}/ca.crt" /var/lib/proxysql/proxysql-ca.pem
 fi
 
 test -e /opt/percona/hookscript/hook.sh && source /opt/percona/hookscript/hook.sh
 
+cat ${PROXY_CFG}
```

Comment:
Do we need to have these two lines ?

Reply:
https://github.com/percona/percona-docker/pull/716/commits/9fc43dd93b9e198370c3f49dcfcb3dd25858bcfd